server/api/users/migrations/0009_user_total_mileage.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.3 on 2023-07-18 06:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0008_alter_user_avatar'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='user',
+            name='total_mileage',
+            field=models.FloatField(default=0.0),
+        ),
+    ]
```

Comment:
I would suggest deleting 0009 and 0011 migrations - merge main into this branch. and then makemigrations again. Main already has a 0009 and 0010.

From my experience on the wais project deleting migration files that has already been on deployed causes problems for production version.